### ISSUE TITLE
Please provide Debian packages

### ISSUE THREAD

**Issue by rharmonson** [neovim/neovim#23512]
### Problem

Debian is very slow to adopt newer versions. Please resume provding .deb for download. I do not nor will I use appimages.

### Expected behavior

Availability of .deb packages for use on stable and testing.

**Comment by clason** [comment]
Why do you think we could do a better and faster job than the actual distro maintainers?

We do not have the resources to provide distro packages on top of our generic builds (which you can use on Debian as well, you know).

**Comment by bfredl** [comment]
Note the `.deb` file we provided, contrary to common myth, was never a "debian package" or an "ubuntu package". you can achieve the same result by downloading the .tar.gz and extract it at any location of your choice, including `/usr/` or `/usr/local/` if you so wish.

**Comment by rharmonson** [comment]
> Why do you think we could do a better and faster job than the actual distro maintainers?
> 
> We do not have the resources to provide distro packages on top of our generic builds (which you can use on Debian as well, you know).

Are you surprised that folks are seeking for .deb files since you previously provided them? I cannot tell if you question is genuine or sarcasm, but the Debian "distro maintainers" do not provide your current release which was why I was downloading the .deb releases from here.

> Note the `.deb` file we provided, contrary to common myth, was never a "debian package" or an "ubuntu package". you can achieve the same result by downloading the .tar.gz and extract it at any location of your choice, including `/usr/` or `/usr/local/` if you so wish.

Thank you. I did not know that information and appreciate you taking the time to provide the guidance. I will rewrite my scripts to use the tar.gz versus the .deb. Very helpful!

**Comment by lewis6991** [comment]
I do not nor will I maintain a deb package.

**Comment by rharmonson** [comment]
You've stopped providing a .deb package as of 0.8.3 and have no intent of proving moving forward.

I got the message loud clear!

**Comment by clason** [comment]
> Are you surprised that folks are seeking for .deb files since you previously provided them? 

Not surprised, but we did make clear why we removed them.

> I cannot tell if your question is genuine or sarcasm, but the Debian "distro maintainers" do not provide your current release which is why I was downloading the .deb releases from here.

It is quite genuine. We do not provide distro packages, which is a _lot_ of work, and involves a lot more than just wrapping our .tar.gz in a `.deb` wrapper -- which is what we did, and stopped doing because we realized that people mistook this for an actual package, leading to actual harm. (And which, incidentally, is also exactly what the appimage is doing. Kneejerk reactions of "I will not use this service you provide" do not do your case any service.)
